casper/src/main/scala/coop/rchain/casper/genesis/contracts/StandardDeploys.scala

```diff
@@ -86,11 +86,16 @@ object StandardDeploys {
       1559156420651L
     )
 
-  def revGenerator(vaults: Seq[Vault], supply: Long): Signed[DeployData] =
+  def revGenerator(
+      vaults: Seq[Vault],
+      supply: Long,
```

Comment:
What is the purpose of `supply`? It's not used anywhere.

Reply:
I'm not really sure TBH. It used set to `Long.MaxValue` everywhere I see it used.

Reply:
My assumption is that was once used for something, but is now deprecated.

Reply:
We will address this later.